### ISSUE TITLE
still can't append message to conversation window

### ISSUE THREAD

**Issue by chy168** [emesene/emesene#122]
Sorry for closing the issue #120.
The problem still exists.

I type some message and "enter", I got:
    Traceback (most recent call last):
      File "/tmp/emesene-emesene-19a44ba/emesene/gui/gtkui/TextBox.py", line  173, in _on_key_press_event
        self.on_send_message(self.text)
      File "/tmp/emesene-emesene-19a44ba/emesene/gui/base  /Conversation.py", line 208, in _on_send_message
        text, self.emcache.parse(), self.emcache.path, self.cstyle, self.first)
    TypeError: send_message() takes exactly 7 arguments (8 given)

I was confirm the regression test from Sbte:
5025a6023cedfeb4d6d3029ff8c17f8316dc28ad is the first bad commit

```
Implemented custom emoticons sending. Tested with amsn, makes emesene1 crash (some weirdness happens). Closes <a href="/emesene/emesene/issues/#issue/96" class="internal">#96</a>

Fixed style in incoming messages in AdiumTextBox.
Possibly broken some output backend with the add of some dict/list here and there.
```

any idea ?

**Comment by c10ud** [comment]
try latest!

**Comment by c10ud** [comment]
Hopefully fix the Textbox out. Please use Adium since it's moar cool. Closed by c1d9810da94ac890c2affd011bc27ed897cc3eb8

**Comment by Sbte** [comment]
Seems to be fixed indeed. Thanks.
